.circleci/config.yml

```diff
@@ -258,18 +263,14 @@ jobs:
       - carthage-bootstrap
       - run:
           name: iOS
-          command: xcodebuild -sdk iphonesimulator -project MapboxDirections.xcodeproj -scheme 'MapboxDirections iOS' -destination 'platform=iOS Simulator,OS=<< parameters.iOS >>,name=<< parameters.device >>' clean build <<# parameters.test >>test<</ parameters.test >><<# parameters.codecoverage >> -enableCodeCoverage "YES"<</ parameters.codecoverage >>
+          command: xcodebuild -sdk iphonesimulator -project MapboxDirections.xcodeproj -scheme 'MapboxDirections iOS' -destination 'platform=iOS Simulator,OS=<< parameters.iOS >>,name=<< parameters.device >>' clean build <<# parameters.test >>test<</ parameters.test >><<# parameters.codecoverage >> -enableCodeCoverage "YES" | xcpretty --report junit --output test-results/junit.xml<</ parameters.codecoverage >>
       - when:
           condition: << parameters.codecoverage >>
           steps:
-            - publish-codecov
+            - publish-coverage
       - run:
           name: tvOS
           command: xcodebuild -project MapboxDirections.xcodeproj -scheme 'MapboxDirections tvOS' -destination 'platform=tvOS Simulator,name=Apple TV,OS=<< parameters.tvOS >>' clean build <<# parameters.test >>test <</ parameters.test >> <<# parameters.codecoverage >>-enableCodeCoverage YES<</ parameters.codecoverage >>
-      - when:
```

Comment:
Could you please explain this change?

Reply:
With codecov, it was possible to report multiple coverage report per one job `build-job`. With the coverage display as artifact in CircleCI, only one test run will be reported, so I reported the coverage only for this iOS run.

<img width="138" alt="Screenshot 2025-02-06 at 12 13 54" src="https://github.com/user-attachments/assets/380b102e-139f-45b6-ad9d-df2ab3f37e91" />

Reply:
But what about other platforms?

Reply:
Currently, coverage is collected only per one platform. Without this pull request, it will be collected per 0 platforms.

Currently, without codecov we don't report the coverage report to the third-party tool. I think it's convenient to see artifacts only per 1 platform because CircleCI artifacts are not designed to display lots of html. If you believe it's critical to collect MacOS and TV OS coverage as well, let's discuss our options.

Reply:
Does it make sense to disable code coverage for all other platforms (currently it is set to`-enableCodeCoverage YES`) to speed up build?